extensions/guacamole-auth-openid/src/main/resources/guac-manifest.json

```diff
@@ -17,14 +17,6 @@
 
     "js" : [
         "openid.min.js"
-    ],
-
-    "css" : [
-        "openid.min.css"
-    ],
-
-    "resources" : {
-        "templates/openidTokenField.html" : "text/html"
-    }
+    ]
```

Comment:
With the template and CSS going away, are similar changes needed for the OpenID `pom.xml`?

The CAS `pom.xml` shows the JS/CSS minification and AngularJS template caching removed. For OpenID, it makes sense that JS minification would stick around, but what about the rest?

Reply:
Ah, I guess template caching can go away, and I can strip out the CSS minification bits.